src/UI/Buyer/src/app/shared/components/line-item-card/line-item-card.component.html

```diff
@@ -3,6 +3,7 @@
     <div class="row align-items-sm-center">
       <div class="col-3 p-0">
         <img class="img-thumbnail"
+             appFallbackImage
              [src]="lineitem.Product.xp?.primaryImageURL || 'http://placehold.it/300x300'">
```

Comment:
Just want to make sure you know  || 'http://placehold.it/300x300'" is still in the src attribute. Is that how you want it?

Reply:
Yep. the `|| 'http://placehold.it/300x300'" ` will take care of the scenario where there is no image defined at all.

My directive will take effect when an image url is defined, but getting that image fails. Possibly the url is simply incorrect. In that case, on fail, itll get replaced by the placeholder url